### ISSUE TITLE
Discuss Browse tabs

### ISSUE THREAD

**Issue by erinchambers** [CDRH/orchid#326]
Flagged by axe DevTools:

> **Certain ARIA roles must contain particular children:** Ensure elements with an ARIA role that require child roles contain them.
> Element has children which are not allowed: a[tabindex]

Here's what it flagged - note that `role="tab"` is on the link, which is not a direct child of the `tablist`.

``` html
<ul class="nav nav-tabs buffer-bottom" role="tablist">
  <li class="" role="presentation">
    <a aria-controls="person.name" role="tab" href="/ardhi/browse/person.name">
    People
    </a>
  </li>
[...]
</ul>
```

**Update:** Turns out this is a [false positive](https://stackoverflow.com/questions/69756534/how-to-fix-role-tablist-role-tab-axe-violations) - see comment in this GitHub issue.

That said... I have a couple questions for us to think about.
  - Tabs are generally meant to be used to dynamically change content without reloading the page; we are reloading the page here, so—regardless of visual appearance—should these really be tabs (with all the ARIA attributes that come with that)? Or should they be a simpler secondary navigation menu (no ARIA craziness, no tabpanels in the body of the page)?
  - ...and in that vein, if they stay as tabs, do they follow the [Tabs Pattern from the WAI-ARIA APG?](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/)
  - Is `role="presentation" serving a purpose here? See related issue: [On Browse tabs, is ARIA role="presentation" necessary? (#332)](https://github.com/CDRH/orchid/issues/332)

**Comment by erinchambers** [comment]
From Stack Overflow answer regarding tabs and axe DevTool's "Certain ARIA roles must contain particular children" flag: 

> Axe is wrong. The [spec for the tab role](https://www.w3.org/TR/wai-aria/#tab) says:
> 
> _Authors MUST ensure elements with role tab are **contained in**, or owned by, an element with the role tablist._
> 
> Emphasis mine. Notice that is says a tab must be contained in a tablist. Contained in does not mean a direct child but rather a descendent.
> 
> This is further emphasized if you look at the next phrase, _or owned by_. "Owned" has a [definition](https://www.w3.org/TR/wai-aria/#dfn-owned-element):
> _An 'owned element' is any DOM descendant of the element, any element specified as a child via aria-owns, or any DOM descendant of the owned child._
> 
> So again, as long as the tab is a descendant of a tablist (child, grandchild, great-grandchild, etc), then technically it's valid and axe is wrong.

They also suggest a workaround: use `aria-owns="{list of tab IDs separated by spaces}"` on the `role="tablist"` element. This keeps axe and other tools from throwing the false positive. Something like this...

```
<ul role="tablist" aria-owns="tab-places tab-people">
  <li>
    <a id="tab_places" aria-controls="person.name" role="tab" href="/ardhi/browse/person.name">Places</a>
  </li>
  <li>
    <a id="tab_people" aria-controls="places" role="tab" href="/ardhi/browse/places">
  </li>
</ul>
```
